### ISSUE TITLE
Restart Apache after generating MapCache config

### ISSUE THREAD

**Issue by rbovard** [camptocamp/c2cgeoportal#1795]
See #1729 for history

I noticed that after a `make clean` and a `make build` I need to restart manually Apache to have access to tiles.

I there an opertation order issue?

````
sudo /usr/sbin/apache2ctl graceful
touch .build/apache.timestamp
.build/venv/bin/c2c-versions CONST_packages.yaml tilecloud-chain
touch .build/test-packages-tilecloud-chain.timestamp
.build/venv/bin/generate_controller --generate-mapcache-config
.build/venv/bin/generate_controller --generate-apache-config
````

**Comment by sbrunner** [comment]
Issue fixed by:
https://github.com/camptocamp/c2cgeoportal/pull/1740

**Comment by rbovard** [comment]
Ok sorry, didn't notice that...

**Comment by sbrunner** [comment]
No problem :-)

**Comment by rbovard** [comment]
@sbrunner Are you sure it's fixed? In `1.6.7rc6` I still need to restart Apache after a clean and build to access my tiles...

**Comment by sbrunner** [comment]
Effectively it's not working, I just do a pull request to fix that :-)
